CHANGELOG.md

```diff
@@ -1,5 +1,86 @@
 [//]: # (current developments)
 
+## 24.1.0 (2024-01-24)
+
+### Enhancements
+
+* Verify signatures on to-be-installed packages instead of on all packages. (#11545, #13053)
+* Add a new `pre-solves` plugin hook. (#13053)
+* Add a new `post-solves` plugin hook. (#13053)
+* Add support for Python 3.12. (#13072)
+* Check `repodata.json.zst` for faster repodata downloads. (#13256)
+* Add `--skip-flexible-search` option to skip flexible search. (#13315)
+* Add type hinting for `VersionOrder` class. (#13380)
```

Comment:
I don't know if this belongs in Enhancements vs Other.

Reply:
I'm ambivalent about it, don't you think it's an enhancement? I guess that section is more about enhancements for "end users"?

Reply:
That's a good point about enhancements for end users; I vote for this to move to the "Other" section